gravitee-apim-console-webui/src/management/application/creation/components/application-creation-form.component.html

```diff
@@ -52,13 +52,19 @@
 
     <mat-form-field>
```

Comment:
Should we hide the form field when there is no group to display and the group is not required ?

https://github.com/user-attachments/assets/c95f82cd-37d7-4a97-ad47-ac0f8133c0fd